runtime/src/cryptographic_mailbox.rs

```diff
@@ -0,0 +1,178 @@
+/*++
+
+Licensed under the Apache-2.0 license.
+
+File Name:
+
+    cryptographic_mailbox.rs
+
+Abstract:
+
+    File contains exports for the Cryptographic Mailbox API commands.
+
+--*/
+
+use crate::Drivers;
+use arrayvec::ArrayVec;
+use caliptra_cfi_derive_git::cfi_impl_fn;
+use caliptra_common::mailbox_api::{
+    CmImportReq, CmImportResp, CmKeyUsage, CmStatusResp, MailboxResp, MailboxRespHeader,
+    CMK_MAX_KEY_SIZE_BITS,
+};
+use caliptra_drivers::CaliptraResult;
+use caliptra_error::CaliptraError;
+use zerocopy::{transmute, FromBytes, Immutable, IntoBytes, KnownLayout};
+
+pub const KEY_USAGE_MAX: usize = 256;
+
+// We have 24 bits for the key ID.
+const MAX_KEY_ID: u32 = 0xffffff;
+
+#[derive(Default)]
+pub struct KeyUsage {
```

Comment:
KeyUsage term seems a little overloaded. The CmKeyUsage in mailbox.rs matches how I would expect the term to use used.

Should this driver be something like `CmStorage` and `CmStorageInfo`?